com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.Debug.cs

```diff
@@ -247,6 +251,58 @@ void RefreshDebug<T>(DebugUI.Field<T> field, T value)
                 widgetList.Add(streamingContainer);
             }
 
+            if (parameters.scenarioBlendingShader != null && parameters.sceneData != null)
+            {
+                HashSet<string> allScenarios = new();
+                foreach (var set in parameters.sceneData.bakingSets)
```

Comment:
Shouldn't we just pick the set to which the loaded scenes belong to?

Reply:
hum i didn't do it because you can load scenes from multiple sets but i guess it makes sense, i could use all the sets containing at least one loaded scenes

Reply:
No you can't with APV data, it will complain at runtime and says you are loading two scenes with different baking sets

Reply:
Unless the sets are _exactly_ the same, but that won't be looking correct either way